Makefile

```diff
@@ -36,7 +36,7 @@ clean-build: ## remove build artifacts
 	rm -fr dist/
 	rm -fr .eggs/
 	find . -name '*.egg-info' -exec rm -fr {} +
-	find . -name '*.egg' -exec rm -f {} +
+	find . -name '*.egg' -exec rm -fr {} +
```

Comment:
Is this relevant? Why do we need this change?

Reply:
yes:
```
$ make install
rm -fr build/
rm -fr dist/
rm -fr .eggs/
find . -name '*.egg-info' -exec rm -fr {} +
find . -name '*.egg' -exec rm -f {} +
rm: cannot remove './env/lib/python3.7/site-packages/python_dateutil-2.7.3-py3.7.egg': Is a directory
rm: cannot remove './env/lib/python3.7/site-packages/six-1.11.0-py3.7.egg': Is a directory
rm: cannot remove './env/lib/python3.7/site-packages/iam_profile_faker-0.3.0-py3.7.egg': Is a directory
rm: cannot remove './env/lib/python3.7/site-packages/text_unidecode-1.2-py3.7.egg': Is a directory
rm: cannot remove './env/lib/python3.7/site-packages/Faker-0.9.0-py3.7.egg': Is a directory
rm: cannot remove './env/lib/python3.7/site-packages/click-6.7-py3.7.egg': Is a directory
```